api/handler/trigger.go

```diff
@@ -79,11 +95,33 @@ func getTrigger(writer http.ResponseWriter, request *http.Request) {
 	if triggerID == "testlog" {
 		panic("Test for multi line logs")
 	}
+
 	trigger, err := controller.GetTrigger(database, triggerID)
 	if err != nil {
 		render.Render(writer, request, err)
 		return
 	}
+
+	if templated := middleware.GetTemplated(request); templated && trigger.Desc != nil {
+		triggerData := moira.TriggerData{
+			ID:   trigger.ID,
+			Name: trigger.Name,
+			Desc: *trigger.Desc,
+		}
```

Comment:
I definitely do not like creation of new trigger here. I propose to change `PopulateDescription`  to return a new instance of trigger with a populated description instead of just string.